code/__DEFINES/cult_defines.dm

```diff
@@ -66,3 +63,7 @@
 #define NARSIE_NEEDS_SUMMONING 2
 #define NARSIE_HAS_RISEN 3
 #define NARSIE_HAS_FALLEN -1
+
+#define GET_CULT_DATA(variable, default) (SSticker.cult_data ? SSticker.cult_data.variable : default)
+
+#define IS_CULTIST(mob) (isliving(mob) && mob?:mind?:has_antag_datum(/datum/antagonist/cultist))
```

Comment:
Any particular reason for making this different from the rest of the is_x checks that we normally do, both in capitalization as well as where it's found?

Reply:
Those are technically all procs, and they're spread out all over the codebase. I made this one into a define, and I slowly want to move over to TG's method of doing these antag checks

![image](https://github.com/ParadiseSS13/Paradise/assets/91113370/7d65ca15-ee55-4d8a-abb1-053b5ec6d994)

Reply:
Yeah, we're all over the place on this right now (traitor's don't even have an `istraitor()`). I think it's fine to just standardize on TG's structure.

Reply:
Sounds good to me